scripts/abort_if_branch_is_master.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -e
+
+abort_if_branch_is_master(){
+  CURRENT_BRANCH=`git rev-parse --abbrev-ref HEAD`
```

Comment:
d'après https://marketplace.visualstudio.com/items?itemName=timonwong.shellcheck, il est préférable d'utiliser une autre syntaxe

```suggestion
  CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
```